server/handlers.go

```diff
@@ -660,7 +663,28 @@ func (s *Server) handleAuthCode(w http.ResponseWriter, r *http.Request, client s
 		return
 	}
 
-	accessToken := storage.NewID()
+	token := &internal.AccessToken{
+		Subject:       authCode.Claims.UserID,
+		Username:      authCode.Claims.Username,
+		Email:         authCode.Claims.Email,
+		EmailVerified: authCode.Claims.EmailVerified,
+		Groups:        authCode.Claims.Groups,
+		ConnectorData: string(authCode.ConnectorData),
+		ConnectorId:   authCode.ConnectorID,
+		Expiry:        authCode.Expiry.Unix(),
+	}
+
+	// The accessToken is build from claims
+	accessToken, err := internal.Marshal(token)
+
+	// Encrypt access token to hide sensible data
```

Comment:
Do we mean `sensible` here?
```suggestion
	// Encrypt access token to hide sensitive data
```